stdlib/test/bit/test_bit.mojo

```diff
@@ -104,6 +104,16 @@ def test_countr_zero_simd():
     assert_equal(countr_zero(var7), SIMD[int64_t, simd_width](62, 64, 0, 62))
 
 
+def test_bit_reverse():
+    assert_equal(bit_reverse(-(2**32)), 4294967295)
+    assert_equal(bit_reverse(-1), -1)
+    assert_equal(bit_reverse(0), 0)
+    assert_equal(bit_reverse(1), -9223372036854775808)
+    assert_equal(bit_reverse(2), 4611686018427387904)
+    assert_equal(bit_reverse(3), -4611686018427387904)
```

Comment:
nit: instead of having these long literals, can you also write these tests in terms of power of two?

Reply:
just fixed the tests. They are now all in power of two now!